__tests__/app.test.js

```diff
@@ -41,4 +41,48 @@ describe('/api/topics', () => {
         expect(response.body.msg).toBe('path not found');
       });
     })
+})
+
+
+
+
+
+
+
+
+
+
+
+describe('/api/articles/:article_id', () => {
+    test('GET: 200 sends a single article object to the client', () => {
+        return request(app)
+        .get("/api/articles/3")
+        .expect(200)
+        .then((response) => {
+            expect(response.body.article.article_id).toBe(3)
+            expect(response.body.article.author).toBe("icellusedkars")
+            expect(response.body.article.title).toBe("Eight pug gifs that remind me of mitch")
+            expect(response.body.article.body).toBe("some gifs")
+            expect(response.body.article.topic).toBe("mitch")
+            expect(response.body.article.votes).toBe(0)
+            expect(response.body.article.created_at).toBe("2020-11-03T09:12:00.000Z")
+            expect(response.body.article.article_img_url).toBe( "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700")
```

Comment:
Nice this will work and I'm not asking you to change it.
But we have some Jest matchers which might be better suited going forward if you wanted to look into them, objectContaining and toMatchObject.